src/models/grant.js

```diff
@@ -107,9 +107,7 @@ export default (sequelize, DataTypes) => {
     recipientInfo: {
       type: DataTypes.VIRTUAL,
       get() {
-        return this.recipient
-          ? `${this.recipient.name} - ${this.number} - ${this.recipientId}`
-          : `${this.number} - ${this.recipientId}`;
```

Comment:
Given that `recipientId` is not nullable, `this.recipient` should always be defined, so the negative case of this ternary can be removed, I think

Reply:
if the Grant.findAll/findOne didn't include model: Recipient, ```this.recipient``` would be null. Not sure if any such actual case exists.

Reply:
Just confirmed with Garrett this this can be null in some cases, so will be reverting this 👍